descriptors/ModuleDescriptor-template.json

```diff
@@ -1091,7 +1091,7 @@
     },
     {
       "id": "service-points",
-      "version": "3.2",
+      "version": "3.3",
```

Comment:
This value should also be changed in the interface documentation - https://github.com/folio-org/mod-inventory-storage/blob/master/ramls/service-point.raml